examples/auto_deploy/paragraf/create_standalone_package.py

```
@@ -316,10 +357,50 @@ def _should_copy(filepath: str) -> bool:
+    basename = os.path.basename(filepath)
+    if ".bak." in basename or basename.endswith("~"):
+        return False
     _, ext = os.path.splitext(filepath)
     return ext in COPY_EXTENSIONS
 
 
+def _tracked_files_under(directory: str) -> list[str]:
+    """Return tracked files below a source directory, with an archive fallback."""
+    relative_directory = os.path.relpath(directory, REPO_ROOT)
+    try:
+        result = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)